portfolio.html

```diff
@@ -0,0 +1,100 @@
+    <title>The Personal Portfolio of Me</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+<nav>
+	<a href="#home" class="nav-item">Home</a>
+	<a href="#projects" class="nav-item">Projects</a>
+	<a href="#about" class="nav-item">About Me</a>
+	<a href="#contact" class="nav-item">Contact</a>
+	<a href="Eleanor Roberts CV.pdf" class="nav-item cv" target="_blank">CV</a>
+</nav>
+	<!-- header -->
+<header id="home">
+	<h1>Eleanor Roberts</h1>
+</header>
+	<!-- projects -->
+<div>
+	<section id="projects">
+		<h2>Projects</h2>
+		<div class="project-item">
+			<img class="project-image" src="nova2.jpeg" alt="Adorable puppy in curtains"/>
+			<div class="project-info">
+				<h3>Project One</h3>
+				<div class="project-description">In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%.</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+		<div class="project-item">
+			<img class="project-image" src="nova3.png" alt="Nova with slipper"/>
+			<div class="project-info">
+				<h3>Project Two</h3>
+				<div class="project-description">This is the description of the second project</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+		<div class="project-item">
+			<img class="project-image" src="Nova1.jpeg" alt="black and white Nova" />
+			<div class="project-info">
+				<h3>Project Three</h3>
+				<div class="project-description">This is the description of the third project</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+	</section>
+</div>
+	<!-- about me -->
+<div>
```

Comment:
Guessing this div is added for formatting reason however if you remove it so you just have the section could you then format the section instead? Center it etc. :)